<!DOCTYPE html
	PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
	<head>
		<title>Jean-Karim Heriche</title>
		<script type="text/javascript" src="js/tabber.js"></script>
		<link rel="stylesheet" type="text/css" href="css/stylesheet.css" />
		<link rel="icon"  type="image/png" href="gfx/favicon.png" />
		<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
		<meta name="Description" content="Jean-Karim H&eacute;rich&eacute;, computational biology, bioinformatics, biological data science">
	</head>
	<meta name="viewport" content="width=device-width"">
	<body>
		<div id="container">
			<div id = "header">
				Jean-Karim H&eacute;rich&eacute;
			</div>
			<div class="tabber">
				<div class="tabbertab" title="Home" id="home">

					<p>I am a staff scientist at the <a href="http://www.embl.de" target="_blank">European Molecular Biology Laboratory</a> (EMBL) working in the <a href="https://www.embl.de/research/units/cbb/index.html">Cell Biology and Biophysics Unit.</a></p>
					<p>I develop and apply data integration and analysis methods to extract biological knowledge from large scale image data sets and multi-omics data. I am also interested in the development of a cloud-based ecosystem for image data analysis.</p>

					<p> Before working for the Cell Biology and Biophysics Unit, I worked in:
						<ul>
							<li><a href="https://www.embl.de/research/units/cbb/ellenberg/index.html" target="_blank">Jan Ellenberg's group</a> at the <a href="http://www.embl.de" target="_blank">EMBL</a> 
							</li>
							<li><a href="http://www.sanger.ac.uk/science/groups/durbin-group" target="_blank">Richard Durbin's group</a> at the <a href="https://www.sanger.ac.uk/" target="_blank">Wellcome Trust Sanger Institute</a> 
							</li>
							<li><a href="http://biochemistry.ucsf.edu/labs/ofarrell/" target="_blank">Pat O'Farrell's group</a> at the <a href="http://biochemistry.ucsf.edu/" target="_blank">University of California, San Francisco</a> (UCSF).
							</li>
						</ul>
					</p>
					
					<p>My current interests revolve around:
						<ul>
							<li>Integration and mining of dynamic biological data</li>
							<ul style="list-style-type:none">
								<li></li>
							</ul>
							<li>Image-based systems biology</li>
							<ul style="list-style-type:none">
								<li></li>
							</ul>
							<li>Bioimage informatics</li>
							<ul style="list-style-type:none">
								<li></li>
							</ul>
							<li>Systems biology of mitosis</li>
							<ul style="list-style-type:none">
								<li></li>
							</ul>
						</ul>
					</p>

					<p>Word cloud representation of PubMed abstracts of my publications (updated 02/2021):
						<p style="text-align:center;"><img src="gfx/word_cloud.png"></p>
					</p>
				</div>

				<div class="tabbertab" title="Publications" id="publications">
					<p>Also on <a href="https://scholar.google.de/citations?user=WI5wH38AAAAJ&hl=en&oi=ao" target="_blank">Google scholar</a></p>
					<ol reversed>

					    <li>Theiss M, <b>H&eacute;rich&eacute; JK</b>, Russell C, Helekal D, Soppitt A, Ries J, Ellenberg J, Brazma A, Uhlmann V. <b>Simulating structurally variable Nuclear Pore Complexes for Microscopy.</b> <a href="https://doi.org/10.1093/bioinformatics/btad587" target="_blank">Bioinformatics, 2023, btad587</a>
					    </li>

					    <li>David R, Rybina A, Burel JM, <b>H&eacute;rich&eacute; JK</b>, Audergon P, Boiten JW, Coppens F, Crockett S, Exter K, Fahrner S, Fratelli M, Goble C, Gormanns P, Grantner T, Gruening B, Gurwitz KT, Hancock J, Harmse H, Holub P, Juty N, Karnbach G, Karoune E, Keppler A, Klemeier J, Lancelotti C, Legras JL, Lister LA, Longo DL, Ludwig R, Madon B, Massimi M, Matser V, Matteoni R, Mayrhofer MT, Ohmann C, Panagiotopoulou M, Parkinson H,; Perseil I, Pfander C, Pieruschka R, Raess M, Rauber A, Richard AS, Romano P, Rosato A, Sanchez-Pla A, Sansone SA, Sarkans U, Serrano-Solano B, Tang J, Tanoli Z, Tedds J, Wagener H, Weise M, Westerhoff HV, Wittner R, Ewbank J, Blomberg N, Gribbon P. <b>"Be Sustainable", Recommendations for FAIR Resources in Life Sciences research: EOSC-Life's Lessons.</b> <a href="https://www.embopress.org/journal/14602075">In press EMBO J.</a>

					    </li>

					    <li>Martinez A, <b>H&eacute;rich&eacute; JK</b>, Calvo M, Tischer C, Otxoa-de-Amezaga A, Pedragosa J, Bosch A, Planas AM, Petegnief V. <b>Characterization of microglia behaviour in healthy and pathological conditions with image analysis tools.</b>  <a href="https://doi.org/10.1098/rsob.220200" target="_blank">Open Biol. 2023. Open Biol.13:220200.</a>
					    </li>

					    <li>Muller C, Serrano-Solano B, Sun Y, Tischer C, <b>H&eacute;rich&eacute; JK</b>. <b>The Image Data Explorer: interactive exploration of image-derived data.</b> <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0273698" target="_blank"> PLOS ONE 17(9): e0273698.</a>
					    </li>

					    <li>Sabinina VJ, Hossain MJ, <b>H&eacute;rich&eacute; JK</b>, Hoess P, Nijmeijer B, Mosalaganti S, Kueblbeck M, Callegari A, Szymborska A, Beck M, Ries J, Ellenberg J. <b>Three-dimensional superresolution fluorescence microscopy maps the variable molecular architecture of the nuclear pore complex.</b> <a href="https://www.molbiolcell.org/doi/10.1091/mbc.E20-11-0728" target="_blank">Mol Biol Cell. 2021 Aug 15;32(17):1523-1533</a>. PMID: 34191541.
					    </li>

					    <li>Sarkans U, Chiu W, Collinson L, Darrow MC, Ellenberg J, Grunwald D,  <b>H&eacute;rich&eacute; JK</b>, Iudin A, Martins GG, Meehan T, Narayan K, Patwardhan A, Russell MRG, Saibil HR, Strambio-De-Castillia C, Swedlow JR, Tischer C, Uhlmann V, Verkade P, Barlow M, Bayraktar O, Birney E, Catavitello C, Cawthorne C, Wagner-Conrad S, Duke E, Paul-Gilloteaux P, Gustin E, Harkiolaki M, Kankaanp&auml;&auml; P, Lemberger T, McEntyre J, Moore J, Nicholls AW, Onami S, Parkinson H, Parsons M, Romanchikova M, Sofroniew N, Swoger J, Utz N, Voortman LM, Wong F, Zhang P, Kleywegt GJ, Brazma A. <b>REMBI: Recommended Metadata for Biological Images-enabling reuse of microscopy data in biology.</b> Nat Methods. 2021 May 21;18(12):1418-1422. PMID: <a href="https://europepmc.org/article/MED/34021280" target="_blank">34021280</a>.
					    </li>

					    <li>Paul-Gilloteaux P, Tosi S, <b>H&eacute;rich&eacute; JK</b>, Gaignard A, M&eacute;nager H, Mar&eacute;e R, Baecker V, Klemm A, Kala&scaron; M, Zhang C, Miura K, Colombelli J. <b>Bioimage analysis workflows: community resources to navigate through a complex ecosystem.</b> <a href="https://doi.org/10.12688/f1000research.52569.1" target="_blank">F1000Research 2021, 10:320</a></li>
						
					    <li>Alladin A, Chaible L, Garcia del Valle L, Reither S, Loeschinger M, Wachsmuth M, <b>H&eacute;rich&eacute; JK</b>, Tischer C, Jechlinger M. <b>Tracking cells in epithelial acini by light sheet microscopy reveals proximity effects in breast cancer initiation.</b> <a href="https://elifesciences.org/articles/54066" target="_blank">eLife. 2020; 9: e54066.</a>
					    </li>
					    
					    <li><b>H&eacute;rich&eacute; JK</b>, Alexander S, Ellenberg J. <b>Integrating imaging and omics: Computational methods and challenges.</b> <a href="https://doi.org/10.1146/annurev-biodatasci-080917-013328" target="_blank">Annu. Rev. Biomed. Data Sci. 2019. 2:1, 175-197</a>
					    </li>

					    <li>Cai Y, Hossain MJ, <b>H&eacute;rich&eacute; JK</b>, Politi AZ, Walther N, Koch B, Wachsmuth M, Nijmeijer B, Kueblbeck M, Martinic M, Ladurner R, Peters JMP, Ellenberg J. <b>An experimental and computational framework to build a dynamic protein atlas of human cell division.</b> Nature. 2018 Sep;561(7723):411-415. doi: 10.1038/s41586-018-0518-z. PubMed PMID: <a href ="http://www.ncbi.nlm.nih.gov/pubmed/30202089" target="_blank">30202089</a>. <a href="https://www.biorxiv.org/content/early/2018/07/17/227751" target="_blank">bioRxiv</a>.
					    </li>
					    
					    <li>Ivanov MP, Ladurner R, Poser I, Beveridge R, Rampler E, Hudecz O, Novatchkova M, <b>H&eacute;rich&eacute; JK</b>, Wutz G, van der Lelij P, Kreidl E, Hutchins JR, Axelsson-Ekker H, Ellenberg J, Hyman AA, Mechtler K, Peters JM. <b>The replicative helicase MCM recruits cohesin acetyltransferase ESCO2 to mediate centromeric sister chromatid cohesion.</b> EMBO J. 2018 Jun 21. pii: e97150. doi: 10.15252/embj.201797150. PubMed PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29930102" target="_blank">29930102</a>	
					    </li>
					    
					    <li>Xiang W, Roberti MJ, <b>H&eacute;rich&eacute; JK</b>, Huet S, Alexander S, Ellenberg J. <b>Correlative live and super-resolution imaging reveals the dynamic structure of replication domains.</b> J Cell Biol. 2018 Mar 23. doi: 10.1083/jcb.201709074. PubMed PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29572382"  target="_blank">29572382</a>. <a href="https://www.biorxiv.org/content/early/2018/01/24/189373" target="_blank">bioRxiv</a>.
					    </li>
					    
					    <li>Otsuka S, Steyer AM, Schorb M, <b>H&eacute;rich&eacute; JK</b>, Hossain MJ, Sethi S, Kueblbeck M, Schwab Y, Beck M, Ellenberg J. <b>Postmitotic nuclear pore assembly proceeds by radial dilation of small ER membrane openings.</b> Nat Struct Mol Biol. 2018 Jan;25(1):21-28. PubMed PMID: <a href ="http://www.ncbi.nlm.nih.gov/pubmed/?term=29323269" target="_blank">29323269</a>. <a href="https://www.biorxiv.org/content/early/2017/09/21/141150" target="_blank">bioRxiv</a>.
					    </li>

					    <li>McMurry JA, Juty N, Blomberg N, Burdett T, Conlin T, Conte N, Courtot M, Deck J, Dumontier M, Fellows DK, Gonzalez-Beltran A, Gormanns P, Grethe J, Hastings J, <b>H&eacute;rich&eacute; JK</b>, Hermjakob H, Ison JC, Jimenez RC, Jupp S, Kunze J, Laibe C, Le Nov&egrave;re N, Malone J, Martin MJ, McEntyre JR, Morris C, Muilu J, M&uuml;ller W, Rocca-Serra P, Sansone SA, Sariyar M, Snoep JL, Soiland-Reyes S, Stanford NJ, Swainston N, Washington N, Williams AR, Wimalaratne SM, Winfree LM, Wolstencroft K, Goble C, Mungall CJ, Haendel MA, Parkinson H. <b>Identifiers for the 21st century: How to design, provision, and reuse persistent identifiers to maximize utility and impact of life science data.</b> PLoS Biol. 2017 Jun 29;15(6):e2001414. PubMed PMID: <a href ="http://www.ncbi.nlm.nih.gov/pubmed/?term=28662064" target="_blank">28662064</a>
					    </li>
					    
					    <li>Serrano-Solano B, D&iacute;az Ramos A, <b>H&eacute;rich&eacute; JK</b>, Ranea JA. <b>How can functional annotations be derived from profiles of phenotypic annotations ?</b> BMC Bioinformatics. 2017 Feb 10;18(1):96. doi: 10.1186/s12859-017-1503-5. PubMed PMID: 28183267; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC5304448"  target="_blank">PMC5304448</a>
					    </li>

					    <li>Pedersen RS, Karemore G, Gudjonsson T, Rask MB, Neumann B,  <b>H&eacute;rich&eacute; JK</b>, Pepperkok R, Ellenberg J, Gerlich DW, Lukas J, Lukas. <b>Profiling DNA damage response following mitotic perturbations.</b> Nat Commun. 2016 Dec 15;7:13887. doi: 10.1038/ncomms13887. PubMed PMID: 27976684; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC5172227"  target="_blank">PMC5172227</a>
					    </li>

					    <li>Jupp S, Malone J, Burdett T, <b>H&eacute;rich&eacute; JK</b>, Williams E, Ellenberg J, Parkinson H, Rustici G. <b>The cellular microscopy phenotype ontology.</b> J Biomed Semantics. 2016 May 18;7:28. doi: 10.1186/s13326-016-0074-0. eCollection 2016. PubMed PMID: 27195102; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC4870745"  target="_blank">PMC4870745</a>
					    </li>

					    <li>Isokane M, Walter T, Mahen R, Nijmeijer B, <b>H&eacute;rich&eacute; JK</b>, Miura K, Maffini S, Ivanov M, Kitajima T, Peters JM, Ellenberg J. <b>ARHGEF17 is an essential spindle assembly checkpoint factor that targets Mps1 to kinetochores.</b> J Cell Biol. 2016 Mar 7;212(6):647-659. doi: 10.1083/jcb.201408089. PubMed PMID: 26953350; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC4792069"  target="_blank">PMC4792069</a>
					    </li>
					    
					    <li> Lees JG, <b>H&eacute;rich&eacute; JK</b>, Morilla I, Fern&aacute;ndez JM, Adler P, Krallinger M, Vilo J, Valencia A, Ellenberg J, Ranea JA, Orengo C.<b> FUN-L: Gene prioritization for RNAi screens.</b> Bioinformatics. 2015 Feb 8. pii: btv073. PubMed PMID: <a href ="http://www.ncbi.nlm.nih.gov/pubmed/?term=25667547" target="_blank">25667547</a>
					    </li>

					    <li> Aleksic J, Alexa A, Attwood TK, Chue Hong N, Dahl&ouml; M, Davey R, Dinkel H, F&ouml;rstner KU, Grigorov I, <b>H&eacute;rich&eacute; JK,</b> Lahti L, MacLean D, Markie ML, Molloy J, Schneider MV, Scott C, Smith-Unna R, Vieira BM; as part of the AllBio: Open Science & Reproducibility Best Practice Workshop.<b> An open science peer review oath.</b> Version 2. F1000Res. 2014 Nov 12 [revised 2015 Jan 9];3:271. doi: 10.12688/f1000research.5686.2. eCollection 2014. PubMed PMID: 25653839; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC4304228" target="_blank">PMC4304228</a>.</li>
					    
					    <li>Patwardhan A, Ashton A, Brandt R, Butcher S, Carzaniga R, Chiu W, Collinson L, Doux P, Duke E, Ellisman MH, Franken E, Gr&uuml;newald K, <b>H&eacute;rich&eacute; JK</b>, Koster A, K&uuml;hlbrandt W, Lagerstedt I, Larabell C, Lawson CL, Saibil HR, Sanz-Garc&iacute;a E, Subramaniam S, Verkade P, Swedlow JR, Kleywegt GJ. <b>A 3D cellular context for the macromolecular world.</b> Nat Struct Mol Biol. 2014 Oct 7;21(10):841-845. doi: 10.1038/nsmb.2897. PubMedID: <a href ="http://www.ncbi.nlm.nih.gov/pubmed/?term=25289590" target="_blank">25289590</a></li>

					    <li> <b>H&eacute;rich&eacute; JK</b>, Lees JG, Morilla I, Walter T, Petrova B, Roberti MJ, Hossain MJ, Adler P, Fern&aacute;ndez JM, Krallinger M, H&auml;ring CH, Vilo J, Valencia A, Ranea JA, Orengo C, Ellenberg J. <b>Integration of biological data by kernels on graph nodes allows prediction of new genes involved in mitotic chromosome condensation.</b> Mol Biol Cell. 2014 Aug 15;25(16):2522-2536. doi: 10.1091/mbc.E13-04-0221. PubMed PMID: 24943848; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC4142622"  target="_blank">PMC4142622</a>
					    </li>
					    
					    <li> <b>H&eacute;rich&eacute; JK</b>.<b> Systematic cell phenotyping.</b> In: JM Hancock (editor). Phenomics. Boca Raton (FL) CRC Press; 2014. Chapter 5.
					    </li>
					    
					    <li> Pau G, Walter T, Neumann B, <b>H&eacute;rich&eacute; JK</b>, Ellenberg J, Huber W. <b>Dynamical modelling of phenotypes in a genome-wide RNAi live-cell imaging assay.</b> BMC Bioinformatics. 2013 Oct 16;14:308. doi: 10.1186/1471-2105-14-308. PubMed PMID:24131777; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC3827932" target="_blank">PMC3827932</a>.
					    </li>
					    
					    <li> Alma&ccedil;a J, Faria D, Sousa M, Uliyakina I, Conrad C, Sirianant L, Clarke LA, Martins JP, Santos M, <b>H&eacute;rich&eacute; JK</b>, Huber W, Schreiber R, Pepperkok R, Kunzelmann	K, Amaral MD. <b>High-content siRNA screen reveals global ENaC regulators and potential cystic fibrosis therapy targets.</b> Cell. 2013 Sep 12;154(6):1390-400. doi: 10.1016/j.cell.2013.08.045. PubMed PMID: <a href ="http://www.ncbi.nlm.nih.gov/pubmed/?term=24034256" target="_blank">24034256</a>.
					    </li>
					    
					    <li> Petrova B, Dehler S, Kruitwagen T, <b>H&eacute;rich&eacute; JK</b>, Miura K, H&auml;ring CH.
						<b>Quantitative analysis of chromosome condensation in fission yeast.</b> Mol Cell Biol.
						2013 Mar;33(5):984-98. doi: 10.1128/MCB.01400-12. Epub 2012 Dec 21. PubMed PMID: 
						23263988; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC3623070" target="_blank">PMC3623070</a>.
					    </li>

					    <li> Gudjonsson T, Altmeyer M, Savic V, Toledo L, Dinant C, Gr&oslash;fte M, Bartkova J,
						Poulsen M, Oka Y, Bekker-Jensen S, Mailand N, Neumann B, <b>H&eacute;rich&eacute; JK</b>, Shearer R,
						Saunders D, Bartek J, Lukas J, Lukas C. <b>TRIP12 and UBR5 suppress spreading of
						chromatin ubiquitylation at damaged chromosomes.</b> Cell. 2012 Aug
						17;150(4):697-709. doi: 10.1016/j.cell.2012.06.039. Epub 2012 Aug 9. PubMed PMID:
						<a href ="http://www.ncbi.nlm.nih.gov/pubmed/?term=22884692" target="_blank">22884692</a>.
					    </li>

					    <li> Simpson JC, Joggerst B, Laketa V, Verissimo F, Cetin C, Erfle H, Bexiga MG,
						Singan VR, <b>H&eacute;rich&eacute; JK</b>, Neumann B, Mateos A, Blake J, Bechtel S, Benes V, Wiemann 
						S, Ellenberg J, Pepperkok R. <b>Genome-wide RNAi screening identifies human proteins
						with a regulatory function in the early secretory pathway.</b> Nat Cell Biol. 2012
						Jun 3;14(7):764-74. doi: 10.1038/ncb2510. PubMed PMID: <a href ="http://www.ncbi.nlm.nih.gov/pubmed/?term=22660414" target="_blank">22660414</a>.
					    </li>
					    
					    <li> Ley S, Weigert A, <b>H&eacute;rich&eacute; JK</b>, Mille-Baker B, Janssen RA, Br&uuml;ne B. <b>RNAi screen 
						in apoptotic cancer cell-stimulated human macrophages reveals co-regulation of
						IL-6/IL-10 expression.</b> Immunobiology. 2013 Jan;218(1):40-51. doi:
						10.1016/j.imbio.2012.01.019. Epub 2012 Feb 2. PubMed PMID: <a href ="http://www.ncbi.nlm.nih.gov/pubmed/?term=22445721" target="_blank">22445721</a>.
					    </li>
					    
					    <li> Bird AW, Erler A, Fu J, <b>H&eacute;rich&eacute; JK</b>, Maresca M, Zhang Y, Hyman AA, Stewart AF. 
						<b>High-efficiency counterselection recombineering for site-directed mutagenesis in 
						    bacterial artificial chromosomes.</b> Nat Methods. 2011 Dec 4;9(1):103-9. doi:
						10.1038/nmeth.1803. PubMed PMID: <a href ="http://www.ncbi.nlm.nih.gov/pubmed/?term=22138824" target="_blank">22138824</a>.
					    </li>
						
					    <li> Moudry P, Lukas C, Macurek L, Neumann B, <b>H&eacute;rich&eacute; JK</b>, Pepperkok R, Ellenberg J,
						Hodny Z, Lukas J, Bartek J. <b>Nucleoporin NUP153 guards genome integrity by
						promoting nuclear import of 53BP1.</b> Cell Death Differ. 2012 May;19(5):798-807.
						doi: 10.1038/cdd.2011.150. Epub 2011 Nov 11. PubMed PMID: 22075984; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC3321618" target="_blank">PMC3321618</a>.
					    </li>	
					    
					    <li> Hegemann B, Hutchins JR, Hudecz O, Novatchkova M, Rameseder J, Sykora MM, Liu 
						S, Mazanek M, L&eacute;n&aacute;rt P, <b>H&eacute;rich&eacute; JK</b>, Poser I, Kraut N, Hyman AA, Yaffe MB,
						Mechtler K, Peters JM. <b>Systematic phosphorylation analysis of human mitotic
						protein complexes.</b> Sci Signal. 2011 Nov 8;4(198):rs12. doi:
						10.1126/scisignal.2001993. PubMed PMID: <a href ="http://www.ncbi.nlm.nih.gov/pubmed/?term=22067460" target="_blank">22067460</a>.
					    </li>

					    <li> Lees JG, <b>H&eacute;rich&eacute; JK</b>, Morilla I, Ranea JA, Orengo CA. <b>Systematic computational
						prediction of protein interaction networks.</b> Phys Biol. 2011 Jun;8(3):035008. doi:
						10.1088/1478-3975/8/3/035008. Epub 2011 May 13. PubMed PMID: <a href ="http://www.ncbi.nlm.nih.gov/pubmed/?term=21572181" target="_blank">21572181</a>.
					    </li>	
						
					    <li> Smedley D, Schofield P, Chen CK, Aidinis V, Ainali C, Bard J, Balling R,
						Birney E, Blake A, Bongcam-Rudloff E, Brookes AJ, Cesareni G, Chandras C, Eppig
						J, Flicek P, Gkoutos G, Greenaway S, Gruenberger M, <b>H&eacute;rich&eacute; JK</b>, Lyall A, Mallon
						AM, Muddyman D, Reisinger F, Ringwald M, Rosenthal N, Schughart K, Swertz M,
						Thorisson GA, Zouberakis M, Hancock JM. <b>Finding and sharing: new approaches to
						registries of databases and services for the biomedical sciences.</b> Database
						(Oxford). 2010 Jul 6;2010:baq014. doi: 10.1093/database/baq014. PubMed PMID:
						20627863; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC2911849" target="_blank">PMC2911849</a>.
					    </li>

					    <li> Neumann B, Walter T, <b>H&eacute;rich&eacute; JK</b>, Bulkescher J, Erfle H, Conrad C, Rogers P,
						Poser I, Held M, Liebel U, Cetin C, Sieckmann F, Pau G, Kabbe R, W&uuml;nsche A,
						Satagopam V, Schmitz MH, Chapuis C, Gerlich DW, Schneider R, Eils R, Huber W,
						Peters JM, Hyman AA, Durbin R, Pepperkok R, Ellenberg J. <b>Phenotypic profiling of 
						the human genome by time-lapse microscopy reveals cell division genes.</b> Nature.
						2010 Apr 1;464(7289):721-7. doi: 10.1038/nature08869. PubMed PMID: 20360735;
						Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC3108885" target="_blank">PMC3108885</a>.
					    </li>
						
					    <li> Hutchins JR, Toyoda Y, Hegemann B, Poser I, <b>H&eacute;rich&eacute; JK</b>, Sykora MM, Augsburg
						M, Hudecz O, Buschhorn BA, Bulkescher J, Conrad C, Comartin D, Schleiffer A,
						Sarov M, Pozniakovsky A, Slabicki MM, Schloissnig S, Steinmacher I, Leuschner M, 
						Ssykor A, Lawo S, Pelletier L, Stark H, Nasmyth K, Ellenberg J, Durbin R,
						Buchholz F, Mechtler K, Hyman AA, Peters JM. <b>Systematic analysis of human protein
						complexes identifies chromosome segregation proteins.</b> Science. 2010 Apr
						30;328(5978):593-9. doi: 10.1126/science.1181348. Epub 2010 Apr 1. PubMed PMID:
						20360068; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC2989461" target="_blank">PMC2989461</a>.
					    </li>

					    <li> Walter T, Shattuck DW, Baldock R, Bastin ME, Carpenter AE, Duce S, Ellenberg 
						J, Fraser A, Hamilton N, Pieper S, Ragan MA, Schneider JE, Tomancak P, <b>H&eacute;rich&eacute;
						JK</b>. <b>Visualization of image data from cells to organisms</b>. Nat Methods. 2010
						Mar;7(3 Suppl):S26-41. doi: 10.1038/nmeth.1431. Review. PubMed PMID: 20195255;
						Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC3650473" target="_blank">PMC3650473</a>.
					    </li>
						
					    <li> O'Donoghue SI, Gavin AC, Gehlenborg N, Goodsell DS, <b>H&eacute;rich&eacute; JK</b>, Nielsen CB,
						North C, Olson AJ, Procter JB, Shattuck DW, Walter T, Wong B. <b>Visualizing
						biological data-now and in the future.</b> Nat Methods. 2010 Mar;7(3 Suppl):S2-4.
						doi: 10.1038/nmeth.f.301. PubMed PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=20195254" target="_blank">20195254</a>.
					    </li>	

					    <li> Walter T, Held M, Neumann B, <b>H&eacute;rich&eacute; JK</b>, Conrad C, Pepperkok R, Ellenberg J. 
						<b>Automatic identification and clustering of chromosome phenotypes in a genome wide
						    RNAi screen by time-lapse imaging.</b> J Struct Biol. 2010 Apr;170(1):1-9. doi:
						10.1016/j.jsb.2009.10.004. Epub 2009 Oct 23. PubMed PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=19854275" target="_blank">19854275</a>.
					    </li>

					    <li> Poser I, Sarov M, Hutchins JR, <b>H&eacute;rich&eacute; JK</b>, Toyoda Y, Pozniakovsky A, Weigl D,
						Nitzsche A, Hegemann B, Bird AW, Pelletier L, Kittler R, Hua S, Naumann R,
						Augsburg M, Sykora MM, Hofemeister H, Zhang Y, Nasmyth K, White KP, Dietzel S,
						Mechtler K, Durbin R, Stewart AF, Peters JM, Buchholz F, Hyman AA. <b>BAC
						TransgeneOmics: a high-throughput method for exploration of protein function in
						mammals.</b> Nat Methods. 2008 May;5(5):409-15. doi: 10.1038/nmeth.1199. Epub 2008
						Apr 6. Erratum in: Nat Methods. 2008 Aug;5(8):748. PubMed PMID: 18391959; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC2871289" target="_blank">PMC2871289</a>.
					    </li>	

					    <li> Ruan J, Li H, Chen Z, Coghlan A, Coin LJ, Guo Y, <b>H&eacute;rich&eacute; JK</b>, Hu Y,
						Kristiansen K, Li R, Liu T, Moses A, Qin J, Vang S, Vilella AJ, Ureta-Vidal A,
						Bolund L, Wang J, Durbin R. <b>TreeFam: 2008 Update.</b> Nucleic Acids Res. 2008
						Jan;36(Database issue):D735-40. Epub 2007 Dec 1. PubMed PMID: 18056084; Europe PMC PMCID: <a href="http://eurpoepmc.org/articles/PMC2238856" target="_blank">PMC2238856</a>.
					    </li>

					    <li> Moses AM, <b>H&eacute;rich&eacute; JK</b>, Durbin R. <b>Clustering of phosphorylation site
						recognition motifs can be exploited to predict the targets of cyclin-dependent
						kinase.</b> Genome Biol. 2007;8(2):R23. PubMed PMID: 17316440; Europe PMC PMCID: 
						<a href="http://europepmc.org/articles/PMC1852407" target="_blank">PMC1852407</a>.
					    </li>

					    <li> Li H, Coghlan A, Ruan J, Coin LJ, <b>H&eacute;rich&eacute; JK</b>, Osmotherly L, Li R, Liu T,
						Zhang Z, Bolund L, Wong GK, Zheng W, Dehal P, Wang J, Durbin R. <b>TreeFam: a
						curated database of phylogenetic trees of animal gene families.</b> Nucleic Acids
						Res. 2006 Jan 1;34(Database issue):D572-80. PubMed PMID: 16381935; Europe PMC
						PMCID: <a href="http://europepmc.org/articles/PMC1347480" target="_blank">PMC1347480</a>.
					    </li>

					    <li> <b>H&eacute;rich&eacute; JK</b>, Ang D, Bier E, O'Farrell PH. <b>Involvement of an SCFSlmb complex in
						timely elimination of E2F upon initiation of DNA replication in Drosophila.</b> BMC
						Genet. 2003 Jun 4;4:9. PubMed PMID: 12787468; Europe PMC PMCID: <a href="http://europepmc.org/articles/PMC161817" target="_blank">PMC161817</a>.
					    </li>
					    
					    <li> <b>H&eacute;rich&eacute; JK</b>, Chambaz EM. <b>Protein kinase CK2alpha is a target for the Abl and
						Bcr-Abl tyrosine kinases.</b> Oncogene. 1998 Jul 9;17(1):13-8. PubMed PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=9671309" target="_blank">9671309</a>.
					    </li>
					    
					    <li> Leroy D, <b>H&eacute;rich&eacute; JK</b>, Filhol O, Chambaz EM, Cochet C. <b>Binding of polyamines to
						an autonomous domain of the regulatory subunit of protein kinase CK2 induces a
						conformational change in the holoenzyme. A proposed role for the kinase
						stimulation.</b> J Biol Chem. 1997 Aug 15;272(33):20820-7. PubMed PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=9252407" target="_blank">9252407</a>.
					    </li>
						
					    <li> <b>H&eacute;rich&eacute; JK</b>, Lebrin F, Rabilloud T, Leroy D, Chambaz EM, Goldberg Y.
						<b>Regulation of protein phosphatase 2A by direct interaction with casein kinase
						    2alpha.</b> Science. 1997 May 9;276(5314):952-5. PubMed PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/?term=9139659" target="_blank">9139659</a>.
					    </li>

					</ol>
				</div>
				<div class="tabbertab" title="Downloads" id="downloads">
					<p>Here are a few projects I've been working on (mostly in R, hosted on <a hraf="https://git.embl.de/">EMBL's GitLab instance):</a>
						<ul>
							<li><a href="https://git.embl.de/heriche/image-data-explorer">The Image Data Explorer</a>: A R/shiny app to plot and explore image-derived data while visualizing the image(s) and regions of interest associated with each data point (see <a href="https://git.embl.de/meechan/image-data-explorer/-/raw/master/screenshots/image_data_explorer_v1.1.screenshot.png">screenshot</a>). 
							</li>
							<li><a href="https://git.embl.de/heriche/lomar">LOMAR</a>: An R package to deal with single molecule localization microscopy data. It implements several point sets registration methods and includes functions to compute similarity between point sets using persistent homology. <a href="https://cran.r-project.org/package=LOMAR">Available on CRAN</a>.
							</li>
							<li><a href="https://git.embl.de/grp-ellenberg/mitotic_cell_atlas">Code to generate the mitotic cell atlas</a>. This is the data processing code, the mitotic cell atlas can be <a href="https://www.mitocheck.org/mitotic_cell_atlas/index.html">visualized here</a>.
							</li>
						</ul>
					<p>Some useful perl modules I have written (hosted on <a href="https://github.com/jkh1/Perlstuff">GitHub</a>):
						<ul>
							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/Matrix.pm">Algorithms::Matrix</a>: an object-oriented matrix manipulation and linear algebra library (requires <a href="https://github.com/ingydotnet/inline-c-pm" target="_blank">Inline::C</a>)</li>
							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/Cube.pm">Algorithms::Cube</a>: an object-oriented library for third order tensors (requires <a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/Matrix.pm">Algorithms::Matrix</a>)</li>
							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/Graph.pm">Algorithms::Graph</a>: extends Graph::Undirected (requires <a href="http://search.cpan.org/~neilb/Graph/lib/Graph/Undirected.pm" target="_blank">Graph::Undirected</a> and <a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/Matrix.pm">Algorithms::Matrix</a>)</li>
							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/FCM.pm">Algorithms::FCM</a>: fuzzy c-means clustering (requires <a href="https://github.com/ingydotnet/inline-c-pm" target="_blank">Inline::C</a>)</li>
							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/RVM.pm">Algorithms::RVM</a>: relevance vector machine for classification (requires <a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/Matrix.pm">Algorithms::Matrix</a>)</li>
							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/HMM.pm">Algorithms::HMM</a>: hidden Markov model (requires <a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/Matrix.pm">Algorithms::Matrix</a>)</li>
							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/GMM.pm">Algorithms::GMM</a>: Gaussian mixture model (requires <a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/Matrix.pm">Algorithms::Matrix</a>)</li>
							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/TimeSeries.pm">Algorithms::TimeSeries</a>: extends Algorithms::Matrix with methods for time series (requires <a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/Matrix.pm">Algorithms::Matrix</a>)</li>
							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/Kernel.pm">Algorithms::Kernel</a>: extends Algorithms::Matrix with methods to compute and manipulate kernels (requires <a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Algorithms/Matrix.pm">Algorithms::Matrix</a>)</li>

							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/HDF5">HDF5</a>: perl wrapper for the <a href="http://www.hdfgroup.org/HDF5/">hdf5 library</a></li>
							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/CellH5">CellH5</a>: perl API to the CellH5 file format (requires <a href="https://github.com/jkh1/Perlstuff/blob/master/modules/HDF5">HDF5</a>)</li>
							<li><a href="https://github.com/jkh1/Perlstuff/blob/master/modules/Labstuff">Labstuff</a>: perl objects to represent various lab items</li>
						</ul>
					</p>
				</div>
				<div class="tabbertab" title="Tutorials" id="tutorials">
				    <p><a href="data_management_guide.html"  target="_blank">Data management guide</a></p>
				    <p><a href="data_management_checklist.html"  target="_blank">Data management checklist</a></p>
				    <p><a href="project_organisation.html"  target="_blank">Project organisation</a></p>
				    <p>Notes from an EMBL workshop on data integration and mining with graphs:
					<ul>
					    <li><a href="data_integration_and_mining_with_graphs-kernels.html" target="_blank">Kernels on graph nodes</a></li>
					    <li><a href="data_integration_and_mining_with_graphs-dynamic_graphs.html" target="_blank">Tensor factorizations for dynamic graphs</a></li>
					</ul>
				    </p>
				</div>
				<div class="tabbertab" title="Links" id="links">
					<P>I am currently involved in the EOSC-Life project:</P>
					<P>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
						<a href="https://www.eosc-life.eu/" target="_blank"><img src="gfx/eosc-life_logo.png" width="128" height="128"></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
					</P>
					<P>I have built and maintain the following projects web sites and databases:</P>
					<P>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
						<a href="http://www.mitocheck.org" target="_blank"><img src="gfx/mitocheck-logo-82.gif"></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
						<a href="http://www.mitosys.org" target="_blank"><img src="gfx/mitosys-logo.png" style="margin-bottom: -1em;"></a>
					</P>
				
					<P>I have been involved in the following projects:</P>
					<P>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
						<a href="http://www.enfin.org" target="_blank"><img src="gfx/enfin_logo.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
						<a href="http://www.systemsmicroscopy.eu/" target="_blank"><img src="gfx/systemsm_logo.png" style="margin-bottom: -1em;"></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
						<a href="http://www.biomedbridges.eu/" target="_blank"><img src="gfx/bmb_logo.png" style="margin-bottom: 0em;"></a>
					</P>

					<P>I have also contributed to the creation of &nbsp;&nbsp;<a href="http://www.treefam.org/"><img style="vertical-align:middle" src="gfx/treefam_logo.png"></a><br><br>
					</P>
					<P>I was a member of the now defunct HUB (Heidelberg Unseminars in Bioinformatics).<br><br>
					</P>
				</div>
				<div class="tabbertab" title="Contact" id="contact">
					<p>
						Jean-Karim H&eacute;rich&eacute;<br>
						Cell Biology & Biophysics Unit<br>
						European Molecular Biology Laboratory<br>
						Meyerhofstra&#223;e 1<br>
						69117 Heidelberg, Germany
					</p>
					<p>
						Phone: +49-6221-387-8677<br>
						Email: heriche (*at*) embl.de<br>
					</p>
					<p>
					</p>
				</div>
			</div>
		</div><!-- End container -->
	</body>
</html>
